about.html

```diff
@@ -0,0 +1,46 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About Us | Autoslug</title>
+    <link rel="icon" href="_assets/logo.png">
+    <link rel="stylesheet" href="homeDesigns.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="stylesheet" href="aboutDesigns.css">
+    <link href="https://fonts.googleapis.com/css2?family=Orbitron:wght@500&family=Sometype+Mono:wght@700&display=swap" rel="stylesheet">
+</head>
+<body style="margin:0px; background-color:rgb(72, 9, 116);">
+    <div class = "topTile" style = "background-color:rgb(31, 10, 45); padding: 0px;">
+        <div class = "topbar">
+            <div class = topbarLeft>
+                <img src="_assets/logo.png" width = "100" height = "100" alt="AutoSlug" style = "padding-left: 10px">
+            </div>
+            <div class = topbarRight>
+                <a href="index.html"class = "links" style = "padding-left: 15px; padding-right: 15px; text-decoration:none">HOME</a>
+                <a href="about.html"class = "links" style = "padding-left: 15px; padding-right: 15px; text-decoration:none">ABOUT</a>
+                <a href="projects.html" class = "links" style = "padding-left: 15px; padding-right: 15px; text-decoration:none">PROJECTS</a>
+                <a href="team.html" class = "links"style = "padding-left: 15px; padding-right: 15px; text-decoration:none">TEAM</a>
+                <a href="donate.html" class = "links"style = "padding-left: 15px; padding-right: 15px; text-decoration:none">DONATE</a>
+            </div>
+        </div>
+    </div>
+    <div class = "centerTile" style = "background-color:rgb(72, 9, 116)">
+        <div style = "color: white;text-align: center;font-family: 'Orbitron', sans-serif;font-size: 50px;padding-bottom: 40px; padding-top: 20px;">ABOUT US</div>
+        <div class = "midbox">
+            <img width = "900px" height = "650px"src="_assets/IMG_2144.JPG">
+            <div style="padding-top: 60px">
+                <div style = "line-height:1.5;font-family: 'Orbitron', sans-serif;font-size:20px; color:white; padding-top: 30px; text-align: center; padding-left: 20px; padding-right: 20px; padding-bottom: 60px;">Autoslug is a Baskin-affiliated organization dedicated to developing practical applications of artificial intelligence (AI), machine learning (ML), and computer vision (CV). We believe that these technologies have the potential to revolutionize many aspects of our lives, and we are committed to providing students with the opportunity to learn how to use them in actual settings. We believe that students should have the opportunity to learn how to use these tools in an applicable real-world environment; to ultimately expand beyond the theoretical side of these advanced concepts and explore their intersection with humans, data, and other forms of hardware.</div>
```

Comment:
now that I'm seeing this in paragraph form, I don't think this is a good font for large amounts of text (especially w/ parentheses, those aren't very readable)